workloads/deployment/workloadDefinition.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: core.oam.dev/v1alpha2
+kind: WorkloadDefinition
+metadata:
+  name: deployments.mycompany.com
+spec:
+  definitionRef:
+    name: ddeployments.mycompany.com
```

Comment:
The name of workloadDefinition should reference to CRD. In this case, I think we should use `deployments.apps`